packages/core/admin/admin/src/hooks/useSettingsForm/index.js

```diff
@@ -94,6 +95,10 @@ const useSettingsForm = (endPoint, schema, cbSuccess, fieldsToPick) => {
           cleanedData.roles = cleanedData.roles.map((role) => role.id);
         }
 
+        if (cleanedData.authenticationDisabled) {
```

Comment:
I think we should rename this `ssoLockedRoles` or `localAuthDisabledRoles` to be more clear.

Reply:
done